ovh/resource_ovh_vrack_dedicated_server.go

```diff
@@ -19,9 +20,22 @@ func resourceVrackDedicatedServer() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"vrack_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ForceNew:      true,
+				DefaultFunc:   schema.EnvDefaultFunc("OVH_VRACK_ID", nil),
+				Description:   "Id of the vrack. DEPRECATED, use `service_name` instead",
+				ConflictsWith: []string{"service_name"},
+			},
+			"service_name": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ForceNew:      true,
+				DefaultFunc:   schema.EnvDefaultFunc("OVH_VRACK", nil),
```

Comment:
Is it intended to have two different env vars `OVH_VRACK_ID` and `OVH_VRACK` here?

Reply:
And shouldn't it be `OVH_VRACK_SERVICE`?